app/src/main/res/layout/fragment_thread.xml

```diff
@@ -133,11 +133,17 @@
             app:menu="@menu/message_menu" />
     </androidx.constraintlayout.widget.ConstraintLayout>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <TextView
         android:id="@+id/emptyView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:gravity="center"
         android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="@string/noConversationSelected"
```

Comment:
You use `attrs` from `ConstraintLayout` whereas you're in a `FrameLayout` ?

Reply:
Oopsy, oversight!